### PR TITLE
5 more common races

### PR DESCRIPTION
**Pull request by Jtecx**
Ping me as well, don't always check my email if there were comments on the pr, but I'll read them here.

### CODE REVIEW

File: src/main/kotlin/xta/game/creature/races/ElfRace.kt [hunk 1]
```diff
@@ -0,0 +1,92 @@
+package xta.game.creature.races
+
+import xta.game.PlayerCharacter
+import xta.game.creature.Race
+import xta.game.creature.RacialStage
+import xta.game.creature.body.*
+import xta.game.stats.Stats
+
+object ElfRace : Race(32, "elf", 5) {
+    var elfHairColour = listOf("black", "leaf green", "golden blonde", "silver")
```
Comment (aimozg) [inline src/main/kotlin/xta/game/creature/races/ElfRace.kt]: In Kotlin use "val" (immutable variable) whenever possible. No need to strive for max immutability, just have a habit to type "val" by default and make it mutable later.
Also, unrelated to this code, but note there is a difference between val/var mutability and mutableList/list

File: src/main/kotlin/xta/game/creature/races/ElfRace.kt [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+    object STAGE_ELF: RacialStage(this, "elf",
+        Stats.STR_MULT to -0.1,
+        Stats.TOU_MULT to -0.15,
+        Stats.SPE_MULT to +0.8,
+        Stats.INT_MULT to +0.8,
+        Stats.WIS_MULT to +0.6,
+        Stats.SENS to + 0.3
+    ){
+        override fun nameOf(creature: PlayerCharacter) = when {
```
Comment (aimozg) [inline src/main/kotlin/xta/game/creature/races/ElfRace.kt]: Not needed, see superclass implementation ("-taur" auto prepended). Same for other stages/races

Comment (aimozg) [reply src/main/kotlin/xta/game/creature/races/ElfRace.kt]: Actually you need to override if racial name should *not* have -taur suffix

File: src/main/kotlin/xta/game/creature/races/ElfRace.kt (hunk 1 above)
Comment (Jtecx) [reply src/main/kotlin/xta/game/creature/races/ElfRace.kt]: Got it, was just me being daft and missing the fact that it said val not var.

Muscle memory.

File: src/main/kotlin/xta/game/creature/races/ElfRace.kt (hunk 2 above)
Comment (Jtecx) [reply src/main/kotlin/xta/game/creature/races/ElfRace.kt]: Okay, should have removed applicable cases, left wolfrace as is cause it checks for gender as well as a suffix if not taur.

**Pull request merged**